spec/features/reservation_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe "reservation actions", :type => :feature, :js => true do
+
+  let(:user){ FactoryGirl.create(:user) }
+
+  it "creates new reservation" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.